content/en/docs/contributing/release-process.md

```diff
@@ -287,7 +287,9 @@ page if a step is missing or if it is outdated.
         example, while releasing `v1.0.0`, you want to compare it with the
         latest pre-released version `v1.0.0-beta.1`:
 
-        <https://github.com/jetstack/cert-manager/compare/v1.0.0-beta.1...master>
+        ```text
+        https://github.com/jetstack/cert-manager/compare/v1.0.0-beta.1...master
```

Comment:
Copied from slack:

[Fixing this is] definitely something we want to do, but I've intentionally filed it away for after the refresh is done just because I'm spinning a lot of plates ATM and don't want to bite anything else off!